config/locales/interface/input_elements/en_emissions_ccus.yml

```diff
@@ -221,12 +221,36 @@ en:
         Energy demand of international aviation is not taken into account in the ETM by default.
         You can change this assumption
         in the <a href="/scenario/demand/transport_international_transport/international-transport">
-        the Demand → Transport → International transport section</a>.
+        International transport</a> section.
         <br/><br/>
         Please note: this slider does not automatically reduce the production of fossil-based
         kerosene in the refineries sector. If you would like to replace fossil kerosene with
-        synthetic kerosene in your scenario, you can reduce the market share of refineries in
-        <a href="/scenario/demand/industry/refineries">the Demand → Industry → Refineries section</a>.
+        synthetic kerosene in your scenario, you can reduce the market share of refineries in the
+        <a href="/scenario/demand/industry/refineries">Refineries</a> section.
+    capacity_of_energy_production_synthetic_kerosene_dispatchable:
+      title: Synthetic kerosene
+      description: |
+        This slider sets the amount of installed dispatchable capacity for synthetic kerosene production 
+        in your scenario. Synthetic kerosene is made from captured CO<sub>2</sub>, hydrogen and electricity. If renewable
+        electricity, green hydrogen and green CO<sub>2</sub> (e.g. Direct Air Capture) are used in
+        this process, it is possible to produce carbon neutral fuels for (e.g.) aviation.
+        <br/><br/> The willingness to pay for dispatchable synthetic kerosene production can be set in the
+        <a href="/scenario/flexibility/flexibility_conversion/conversion-to-kerosene-for-industry">
+        Flexibility</a> section.
```

Comment:
This addition is good, but I would elaborate a bit further: what makes it dispatchable is that it responds to to electricity. It therefore depends on the electricity price and the willingness to pay how much the dispatchable unit produces. Where relevant we can also link to the (new) pages in the documentation.